util/dockerfiles/1.13.1/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM debian:testing
```

Comment:
It seems reasonable to release with a stable release, e.g. `jessie`. This would still be the stable release for 1.14. We might have more elaborate docker image support by 1.15, and release testing / unstable images as well.